packages/gamut-labs/src/Avatar/index.tsx

```diff
@@ -62,9 +62,9 @@ export type AvatarBaseProps = {
   disableDropshadow?: boolean;
 
   /**
-   * Size of the Avatar; small = 32x32, medium = 118x118
+   * Size of the Avatar.
    */
-  size?: 'small' | 'medium';
+  size?: 32 | 48 | 64 | 118;
```

Comment:
Let's add system props to img here so we can also do some media query syntax.

```tsx
import { ResponsiveProp } from '@codecademy/variance';
import { system, styledOptions } from '@codecademy/gamut-styles';

const Image = styled('img', styledOptions<'img'>())(system.layout);
```

```suggestion
  size?: ResponsiveProp<32 | 48 | 64 | 118>;
```